src/app/components/media/MediaTypeDisplayIcon.js

```diff
@@ -11,6 +11,42 @@ import Public from '../../icons/public.svg';
 import Twitter from '../../icons/twitter.svg';
 import YouTube from '../../icons/youtube.svg';
 
+export function mediaTypeFromUrl(url) {
+  let mediaType = 'Link';
+
+  if (/^(https?:\/\/)?((www\.)?youtube\.com|youtu\.be)\/.+$/.test(url)) {
+    mediaType = 'Youtube';
+  } else if (/^(https?:\/\/)?((www\.)?instagram\.com)\/.+$/.test(url)) {
+    mediaType = 'Instagram';
+  } else if (/^(https?:\/\/)?((www\.)?twitter\.com)\/.+$/.test(url)) {
+    mediaType = 'Twitter';
+  } else if (/^(https?:\/\/)?((www\.)?facebook\.com)\/.+$/.test(url)) {
+    mediaType = 'Facebook';
+  }
+
+  return mediaType;
+}
+
+// FIXME Get supported file extensions from backend
+export function mediaTypeFromFilename(fileName) {
+  let mediaType = null;
+
+  const match = fileName.match(/\.([a-zA-Z0-9]+)$/);
+  const fileExtension = match && match[1];
+
+  if (fileExtension) {
+    if (['jpg', 'jpeg', 'JPG', 'JPEG'].includes(fileExtension)) {
```

Comment:
This might not even be relevant here after you address the FIXME but something like

`['jpg', 'jpeg'].includes(fileExtension.toLowerCase())`

would probably be better since it would catch something like `.Jpg` as well, plus it makes the array clearer.

Reply:
Yeah, I totally meant to do that. This was just to get the feature started - fixed it!